core_lib/src/structure/layer.cpp

```diff
@@ -213,72 +213,46 @@ bool Layer::removeKeyFrame(int position)
     return true;
 }
 
-bool Layer::moveKeyFrameForward(int position)
+bool Layer::moveKeyFrame(int position, int offset)
 {
-    return swapKeyFrames(position, position + 1);
-}
+    bool moved = false;
+    if (position == 1 && offset < 0) { return false; }
 
-bool Layer::moveKeyFrameBackward(int position)
-{
-    if (position != 1)
-    {
-        return swapKeyFrames(position, position - 1);
+    if (swapKeyFrames(position, position + offset)) {
+        moved = true;
+    } else {
```

Comment:
Can you change this if else to use the more standard bracket format please.

Reply:
Done